qiskit/visualization/matplotlib.py

```diff
@@ -406,11 +406,29 @@ def _get_layer_widths(self):
                 if self._cregbundle and node.cargs and not isinstance(op, Measure):
                     self._cregbundle = False
                     warn(
-                        "Cregbundle set to False since an instruction needs to refer"
+                        "cregbundle set to False since an instruction needs to refer"
                         " to individual classical wire",
                         RuntimeWarning,
                         2,
                     )
+                if (
```

Comment:
This snippet looks very similar to what's in `latex.py`, would it make sense to move this out to a utils file and reduce duplication of logic?

Reply:
Actually basically matches the 'text' drawer code as well, though it handles the `cargs` version differently. Could definitely be added to utils. Will look into it.

Reply:
@javabster Moved the `cregbundle` checks to `utils.py` in https://github.com/Qiskit/qiskit-terra/pull/7653/commits/829ded3621a87d63b7e4dca8fb0aa180f65e15c1.